src/lib/parquet/src/Flow/Parquet/ParquetFile/RowGroupBuilder/ColumnChunkStatistics.php

```diff
@@ -63,12 +63,12 @@ public function add(string|int|float|null|array|bool|object $value) : void
 
     public function avgStringLength() : int
     {
-        return (int) \ceil($this->totalStringLength / $this->notNullCount());
+        return (int) \ceil($this->totalStringLength / ($this->notNullCount() ?: 1));
```

Comment:
if not null count is 0 it should just return zero, if statement above would be probably cleaner

Reply:
Solved.